contracts/contracts/GrantRound.sol

```diff
@@ -0,0 +1,134 @@
+pragma solidity ^0.8.5;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "./GrantRegistry.sol";
+
+contract GrantRound {
+  using SafeERC20 for IERC20;
+  // @notice Unix timestamp of the start of the round
+  uint256 public startTime;
+  // @notice Unix timestamp of the end of the round
+  uint256 public endTime;
+  // @notice Contract owner
+  address public owner;
+  // @notice GrantsRegistry
+  address public registry;
+  // @notice ERC20 token that accepts pool donations
+  IERC20 public donationToken;
+  // @notice URL pointing to grant round metadata (for off-chain use)
+  string public metaPtr;
+  // @notice minimum contribution amt that can be made
+  uint256 public minContribution;
+  // @notice Set to true if grant round has ended and payouts have been released
+  bool public hasPaidOut;
+
+  /// @notice Emitted when a grant receives a donation
+  event DonationSent(uint96 indexed id, address indexed token, uint256 amount, address dest, address indexed donor);
+
+  modifier onlyOwner() {
```

Comment:
nit: can we move modifiers to the end of the contract? Really I don't care where they are, as long as we're consistent, but generally I've seen them placed at the end by convention. Anyone have strong feelings here?